docs/developer-docs/integrations/sns/launching/launch-summary.md

```diff
@@ -0,0 +1,332 @@
+| Treasury account on the SNS Ledger                | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+
+
+- #### Step 10: SNS participants participate in the decentralization swap.
+
+  When the swap starts, the swap canister holds the number
+  of SNS tokens that were specified. End users can
+  participate in the decentralization swap by transferring ICP tokens to the
+  swap canister.
+
+  
+
+- #### Step 11: SNS canisters become SNS DAO.{#SNS-launch-step-genesis}
+  
+  When the decentralization swap ends, it is first established whether
+  it was successful, e.g., enough ICP have been collected. If the swap was successful,
+  the exchange rate is determined and all SNS tokens are given to the swap participants in
+  neurons. 
+  
+  Currently, this process needs to be triggered by a call to the swap canister that anyone
+  can make.
+  
+  Once all neurons are created, the SNS should be under decentralized control
+  and the pre-decentralization-swap mode is reverted.
+  Thus, the governance canister is set to be fully functional.
+  If the swap is not successful, the decentralization attempt failed and everything
+  is reverted to the state before the SNS launch attempt, including that the dapp’s control
+  is handed back to the original developers of the dapp, and the
+  collected ICP are refunded to the swap participants.
+
+
+If successful, at the end of stage, we have:
+
+|  Decentralization swap state                      | COMMITTED                               |  
+|-------------------                                | ----------------                        |
+
+
+|  Objects in an app subnet                                      | State                    |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| A dapp                                            | Operational                           | ~~Dapp developer principal~~ ~~Newly-created SNS canisters~~ SNS DAO | 
+
+|  Objects in the SNS subnet                                       | State                  |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| ~~A principal that can deploy to SNS subnet~~     | Revoked because it is 1-time use      | NA                        |
+| Newly-created SNS canisters on the SNS subnet     | Normal mode                           | ~~Initial developer neurons~~ SNS DAO |  
+| Initial developer neurons                         | Normal mode                           | Dapp developer principal  |
+| Treasury account on the SNS Ledger                | Normal mode                           | ~~Newly-created SNS canisters on the SNS subnet~~ SNS DAO   |
```

Comment:
while this is true, we also don't rename this 3 lines up. 
These are still the same cansiters, just I would probably use the same name